src/http-handler.ts

```diff
@@ -386,7 +386,7 @@ export class HttpHandler {
 
             if (Utils.isCachingChannel(channel)) {
                 this.server.cacheManager.set(
-                    `app:${appId}:channel:${channel}:cache_miss`,
+                    `app:${appId}:channel:${channel}:cache`,
```

Comment:
I would rather keep the key the same. Existing users that use it already and upgrade to the version containing this change might run into cache_miss not being fired because of the new changed key.

Reply:
I have added a commit that keeps the key as cache_miss.